### DISCUSSION TITLE
SONOFF Zigbee 3.0 USB Dongle Plus-P

### DISCUSSION THREAD

**Discussion by adrianlanzi** [Koenkk/zigbee2mqtt#16939] [General]
Is it possible to plug a SONOFF Zigbee Dongle to the usb port of the internet povider router? I've got a Huawei EchoLife eg8145v5. Just wondering if i can avoid buying a Raspeberry Pi at the beggining

**Comment by freeDom-** [comment]
You can plug that stick into your routers USB, but you won't be able to install Zigbee2MQTT and an MQTT-Broker like Mosquitto on your router. Both are required for the stick to do anything useful and to be able setting up your Zigbee devices.
